satpy/modifiers/spectral.py

```diff
@@ -95,7 +95,7 @@ def _get_tb13_4_from_optionals(optional_datasets):
         return tb13_4
 
     @staticmethod
-    def _get_sun_zenith_from_provided_data(projectables, optional_datasets):
+    def _get_sun_zenith_from_provided_data(_nir, optional_datasets, dtype):
```

Comment:
just nitpicking here, but I would rename `_nir`to `nir` here.

Reply:
Renamed in https://github.com/pytroll/satpy/pull/2642/commits/b992edb6e774c2a9e1dfa35426a54df9c7266492